Assets/Scripts/Player.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Player : MonoBehaviour
+{
+
+    [Header("Horizontal Movement")]
+    public float moveSpeed = 10f;
+    public Vector2 direction;
+    private bool facingRight = true;
+
+    [Header("Components")]
+    public Rigidbody2D rb;
+
+
+    [Header("Physics")]
+    public float maxSpeed = 12f;
+
+    //[Header("Collision")]
+
+
+    // Start is called before the first frame update
+    void Start()
+    {
+
+
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        float horizontalDirection = Input.GetAxisRaw("Horizontal");
+        float verticalDirection = Input.GetAxisRaw("Vertical");
+        direction = new Vector2(horizontalDirection, verticalDirection);
+    }
+
+    void movePlayer(float horizontal)
+    {
+        rb.AddForce(Vector2.right * horizontal * moveSpeed);
```

Comment:
I think it's better to make more intermediate variables since class type matters so much. Since I don't really know these methods, I think it would be better if the variables could just tell me what argument the method needs.

```suggestion
        float horizontalForce = Vector2.right * horizontal * moveSpeed;
        rb.AddForce(horizontalForce);
```

Reply:
Case in point, it's actually a `Vector2` not a `float`.